columnflow/tasks/framework/mixins.py

```diff
@@ -370,6 +375,18 @@ def calibrator_insts(self) -> list[Calibrator]:
             self._calibrator_insts = self.get_calibrator_insts(self.calibrators, {"task": self})
         return self._calibrator_insts
 
+    @property
+    def calibrators_repr(self) -> str:
+        """
+        Return a string representation of the calibrators.
+        """
+        part = "none"
```

Comment:
Could you name this differently, e.g. `r` or so? The `part` might be misleading (although it made sense where it previously came from ;) ). Same below.